code/game/gamemodes/changeling/changeling.dm

```diff
@@ -289,7 +289,8 @@ var/list/slot2type = list("head" = /obj/item/clothing/head/changeling, "wear_mas
 	var/changelingID = "Changeling"
 	var/geneticdamage = 0
 	var/isabsorbing = 0
-	var/geneticpoints = 10
+	var/geneticpoints = 5 //Currency used for purchasing powers - gained from absorption
+	var/total_genetic_points = 5 //Used in respeccing to reset points to the proper amount

```

Comment:
It does, absorbing increases this as well and resets to it(or it should, looking at the code)